README.md

```diff
@@ -78,3 +65,7 @@ For more information, regarding usage in different languages. Visit [Bench-Route
 
 - Instant messaging: [bench-routes/community](https://gitter.im/bench-routes/community)
 - Discussions (Recent): https://groups.google.com/forum/#!forum/bench-routes-discussion
+   
+### Supported by
+
+![JetBrains](https://user-images.githubusercontent.com/36980003/111029750-5b46fc80-8424-11eb-800f-ad7f64844f0d.png)
```

Comment:
This is perfect. Just last one thing, it should be wrapped with a URL that points to jetbrains (use this link: https://jb.gg/OpenSource).